primitives/header-chain/src/lib.rs

```diff
@@ -77,18 +77,35 @@ pub trait FinalityProof<Number>: Clone + Send + Sync + Debug {
 	fn target_header_number(&self) -> Number;
 }
 
-/// Find header digest that schedules next GRANDPA authorities set.
-pub fn find_grandpa_authorities_scheduled_change<H: HeaderT>(
-	header: &H,
-) -> Option<sp_finality_grandpa::ScheduledChange<H::Number>> {
-	let id = OpaqueDigestItemId::Consensus(&GRANDPA_ENGINE_ID);
+pub trait ConsensusLogReader {
+	const ID: ConsensusEngineId;
```

Comment:
Is this const used (or planned to be) used anywhere? I wonder why do we have similar const in both `Engine` trait and here?

Reply:
Same question for `type ConsensusLog` - is it used anywhere?

Reply:
I removed them. They are not used.